lib/src/main.dart

```diff
@@ -218,3 +219,26 @@ class MainPage extends StatelessWidget {
     );
   }
 }
+
+/// When thumbnail is available draw image using full width available.
+/// 
+/// TODO: replace thumbnail with hi-res image.
+Widget addImage(Submission submission) {
+  return submission.thumbnail.toString() != 'self'
+      ? LayoutBuilder(
+          builder: (a, b) => CachedNetworkImage(
+                width: b.biggest.width,
+                imageUrl: submission.thumbnail.toString(),
+                fit: BoxFit.fitWidth,
+                placeholder: (context, url) => CircularProgressIndicator(),
+                errorWidget: (context, url, error) => Text(
+                  'Image not available',

```

Comment:
Can we just display nothing if it's an error widget? I feel like the `Image not available` isn't clean. Just return an empty `SizedBox()`

Reply:
Done.